src/components/common/Toolbar/ToolbarAvatar.tsx

```diff
@@ -38,8 +47,24 @@ export function ToolbarAvatar({ member, title, children }: ToolbarAvatarProps) {
   const displayName = getUserDisplayName(member.userId);
   const avatarUrl = getUserAvatarUrl(member.userId);
 
-  const label =
-    member.userId === ownUserId
+  const label = presenter
```

Comment:
Using the ternary conditional operator here make this code hard to understand. I think that something like what I show below is a lot more readable:

```
const isPresenter = presenter && member.userId === ownUserId;
const isOwnUser = member.userId === ownUserId;

let label;

if (isPresenter) {
  label = t('toolbar.toolbarAvatar.labelPresenting', '{{displayName}} (You) are presenting', {
    displayName,
    context: 'you',
  });
} else if (isOwnUser) {
  label = t('toolbar.toolbarAvatar.label', '{{displayName}} (You)', {
    displayName,
    context: 'you',
  });
} else {
  label = t('toolbar.toolbarAvatar.labelPresenter', '{{displayName}} is presenting', {
    displayName,
  });
}

```

Reply:
I have changed outer ternary operator to if, it makes it a bit better to read

Reply:
I went and refactored a bit more.